app/models/user.rb

```diff
@@ -3,6 +3,9 @@ class User < ApplicationRecord
   has_many :comments, foreign_key: 'author_id'
   has_many :likes, foreign_key: 'author_id'
 
+  validates :name, presence: true, length: { in: 1..250 }
+  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
   def last_three_posts
     posts.last(3)
   end
```

Comment:
- [ ] As the previous reviewer suggested that there is no unit for the **last_three_posts** method in your user_spec file :fearful: Kindly make a unit test and use **last_three_posts** method in that unit test :+1: